# This message expresses syncronized joint and link state in a single message

Header header

sensor_msgs/JointState joint_state

LinkState link_state

